test/cmdlineargs.jl

```diff
@@ -783,6 +783,17 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
             "Int(Base.JLOptions().fast_math)"`)) == JL_OPTIONS_FAST_MATH_DEFAULT
     end
 
+    let JL_OPTIONS_TASK_TIMING_OFF = 0, JL_OPTIONS_TASK_TIMING_ON = 1
+        @test parse(Int,readchomp(`$exename -E
+            "Int(Base.JLOptions().task_timing)"`)) == JL_OPTIONS_TASK_TIMING_OFF
+        @test parse(Int, readchomp(`$exename --task-timing -E
+            "Int(Base.JLOptions().task_timing)"`)) == JL_OPTIONS_TASK_TIMING_ON
```

Comment:
Do you think there's a chance we'll ever want to enable this by default?

And if so, do you think this should be an on/off flag like `--task-timing=on`, rather than `--task-timing`, so that the meaning of the flag can stay unchanged even if we change the defaults in the future? Though i guess we could always introduce a `--no-task-timing` flag in that case in the future.

Reply:
i wasn't sure whether to prefer `--task-timing` or `--task-timing=yes` but i see the second option leaves us a little more room for flexibility so i'll switch to that